public/js/categories.js

```diff
@@ -0,0 +1,6 @@
+const catResult = document.querySelector('.catResult');
+
+catResult.addEventListener('click', (e) => {
+  const name = e.target.id;
+  window.location = `/business?name=${name}`;
```

Comment:
tell eslint that document and window are global variables 
/* global document window */